### PR TITLE
fix: avoid serialization issue with psycopg2

### PR DESCRIPTION
**Pull request by majorgreys**
Ignore this as this is part of an investigation into a bug.

### CODE REVIEW

**Review by jd — CHANGES_REQUESTED** [review]
what issue? description? test? 😢

**Comment by majorgreys** [comment]
This is a potential mitigation that is blocked on feedback from a user which I have had difficulty reproducing with a test case. Hoping that with confirmation of the mitigation we can work out a scenario for reproducing it.

File: .circleci/config.yml [hunk 1]
```diff
@@ -830,13 +822,16 @@ jobs:
       - store_artifacts:
           path: /tmp/docs
 
-  deploy_docs:
```
Comment (brettlangdon) [inline .circleci/config.yml:833]: if you are removing this, can you move to it's own PR?

File: scripts/build-dist-cp27 [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```
Comment (brettlangdon) [inline scripts/build-dist-cp27:1]: these changes should be in their own PR.

also, do we still need this workflow? we get wheel building in GitHub Actions, and we have means of publishing dev builds to PyPI, we shouldn't need manual building/artifacts from CircleCI?

File: ddtrace/contrib/psycopg/connection.py [hunk 3]
```diff
@@ -50,7 +51,7 @@ def execute(self, query, vars=None):  # noqa: A002
             if not s.sampled:
                 return super(TracedCursor, self).execute(query, vars)
 
-            s.resource = query
+            s.resource = stringify(query)
```
Comment (brettlangdon) [inline ddtrace/contrib/psycopg/connection.py:54]: this change looks fine, but the other stuff needs to be separated out.

**Review by brettlangdon — CHANGES_REQUESTED** [review]

**Comment by brettlangdon** [comment]
is this meant to be merged into `0.44` as a backport?

**Comment by majorgreys** [comment]
@brettlangdon, sorry please ignore this PR. It's to isolate a problem and I'll close it when I've built the wheel.

**Pull request closed without merging**